lessons/Algebra2/logarithmic1-exploring-wealth-health/langs/en-us/index.adoc

```diff
@@ -65,7 +71,7 @@ What graphs did you draw? Why do you think your graph accurately reflects the re
 
 @slidebreak
 
-For many of your graphs, the first $40,000 or so of income shows a huge improvement in life expectancy: this might be enough to afford a place to live, see a doctor every once in a while, etc. But will that same $40,000 makes a difference in the life-expectancy of a billionaire?
+For many of your graphs, the first $20,000 or so of income shows a huge improvement in life expectancy: this might be enough to afford a place to live, see a doctor every once in a while, etc. But will that same $20,000 makes a difference in the life-expectancy of a billionaire?
```

Comment:
I was expecting the question, "Does that $20,000 make the same difference in every country on the planet?"